temp_track.js

```diff
@@ -0,0 +1,65 @@
+/* shelly model Shelly 2PM Gen3s
+ */
+
+let CONFIG = {
+  scanInterval_ms: 5000, // miliseconds: read the sensors on every 5000 ms
+  tempThreshold: 3, // °C
+  switch: { id: 0, name: "heater" },
+};
+
+function tempTracker() {
+  try {
+    Shelly.call(
+      "Switch.GetStatus",
+      { id: CONFIG.switch.id },
+      function (result, err_code, err_message) { 
+        if (result !== undefined && err_code === 0) {
+          console.log(
```

Comment:
This log is useful for debugging, but having it in production code is unnecessary. Please, put a conditional debug flag in CONFIG (false by default), which controls whether to print such debug logs.